### PR TITLE
Update KeeperFX, remove KeeperFX Unofficial

### PR DESCRIPTION
**Pull request by hansejo**
Progress has restarted under the original project. This change reflects this

### CODE REVIEW

File: games/k.yaml [hunk 1]
```diff
@@ -88,27 +88,14 @@
...
   license:
   - As-is
-  development: halted
+  development: active
   originals:
   - Dungeon Keeper
   repo: https://github.com/dkfans/keeperfx
   type: remake
-  updated: 2020-05-18
+  updated: 2020-08-25
```
Comment (cxong) [inline games/k.yaml:96]:
```suggestion
  updated: 2022-04-03
```

**Review by cxong — COMMENTED** [review]
also it would be good to add some images/video; check the netlify deploy for what it will look like

File: games/k.yaml (hunk 1 above)
Comment (hansejo) [reply games/k.yaml]: Whoops, thought updated meant something different thanks!

**Comment by hansejo** [comment]
@cxong  I would like to add a screenshot, but I haven't built it yet. And no other screenshots I can find.

**Comment by hansejo** [comment]
@cxong Maybe we shouldn't consider this an engine remake. Main website says that this converts the original DK into a DLL which it then loads. So this is more akin to a patch. Thoughts on this?

File: games/k.yaml [hunk 2]
```diff
@@ -88,27 +88,14 @@
   - C
   license:
   - As-is
-  development: halted
+  development: active
   originals:
   - Dungeon Keeper
   repo: https://github.com/dkfans/keeperfx
   type: remake
```
Comment (cxong) [inline games/k.yaml:95]:
```suggestion
  type: tool
```

File: games/k.yaml [hunk 3]
```diff
@@ -88,27 +88,14 @@
...
-  updated: 2020-05-18
+  updated: 2022-04-02
   url: https://lubiki.keeperklan.com/html/dk_keeperfx.php
```
Comment (cxong) [inline games/k.yaml:97]:
```suggestion
  url: https://lubiki.keeperklan.com/html/dk_keeperfx.php
  video:
    youtube: JwDdLynZPag
```

**Pull request merged**